ocs_ci/utility/utils.py

```diff
@@ -1796,3 +1797,28 @@ def skipif_ocs_version(expressions):
         skip_this = skip_this and eval(comparision_str)
     # skip_this will be either True or False after eval
     return skip_this
+
+
+def get_ocs_version_from_tag(tag):
+    """
+    Parse major.minor version from OCS image tag.
+
+    Args:
+        tag (str): version tag from the image
+
+    Returns
+        str: Version in x.y format
+
+    Raises:
+        ValueError: In case tag which we cannot parse passed.
+
+    """
+    try:
+        version = tag.split(':')[1].lstrip("latest-")
```

Comment:
can we remove this

Reply:
this value is used in log

Reply:
no. this *should* be removed. it is overwritten in the next line... ;-)

Reply:
Not sure how it got deleted but I am almost sure I had updated line bellow to what is coming in next commit

Reply:
If we want log the error in str format in https://github.com/red-hat-storage/ocs-ci/pull/1653/commits/c8f7641eac7ba52d201835347bd44d6861e1e98d#diff-68bc5ed905b5a4cefbac287573f4f60fR1823 , then we need to take this into another variable.

Reply:
Why? if exception happen we just have the string from line 1817 not the version with Version.coerce

Reply:
agreed